recipe/meta.yaml

```diff
@@ -11,7 +11,6 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win]
```

Comment:
If you require C++11, then this would be `skip: true  # [win and py<35]` (note two spaces before selectors).